src/withFacets.js

```diff
@@ -118,8 +119,14 @@ function withFacets(WrappedComponent) {
       const cqlQuery = buildQuery(query, {}, { ...data, query }, { log: () => null }) || '';
       const facetName = facetToOpen || onMoreClickedFacet || focusedFacet;
       const facetNameToRequest = FACETS_TO_REQUEST[facetName];
+      const paramsUrl = new URLSearchParams(window.location.search);
+      const queryIndex = paramsUrl.get('qindex');
 
-      if (cqlQuery) params.query = cqlQuery;
+      if (cqlQuery && queryIndex === browseModeOptions.CALL_NUMBERS) {
```

Comment:
May we try to do in this way? 
`params.query = cqlQuery && queryIndex === browseModeOptions.CALL_NUMBERS ? 'callNumber=""' :  cqlQuery`

Reply:
Ye, definitely I will try when env will be available.

Reply:
I can't do it in this way, because I need to set params. query to cqlQuery only if it presents. In other way, it should be default